GitHub Actions fails to build docs website (`cannot touch 'docs/.nojekyll'`)

The docs build has been failing for a while:

https://github.com/ber-data/bertron-schema/actions/runs/15596239172


```
touch docs/.nojekyll
touch: cannot touch 'docs/.nojekyll': No such file or directory
make: *** [Makefile:219: gendoc-gh] Error 1
Error: Process completed with exit code 2.
```

Should line 46 of `justfile` be pointing to `src/docs` instead of `docs`?

`docdir := "docs"`

--- a/Makefile
+++ b/Makefile
@@ -216,8 +216,8 @@ gendoc: $(DOC_DIR)  ## generate Markdown documentation locally
 	cp $(SRC_DIR)/docs/js/*.js $(DOC_DIR)/javascripts/
 
 gendoc-gh: ## generate HTML documentation for deployment on GitHub Pages
-	touch $(DOC_DIR)/.nojekyll
 	make gendoc
+	touch $(DOC_DIR)/.nojekyll
 	make mkd-gh-deploy
 
 testdoc: gendoc serve

